### ISSUE TITLE
Prometheus/Grafana Deprecated Helm Repo References

### ISSUE THREAD

**Issue by jgrossmac** [solo-io/gloo#4003]
**Describe the bug**

When trying to download the gloo-ee Helm chart locally, and then deploy it to my cluster, I get a deprecation error notice that prevents me from proceeding.  Running `helm install ...` straight away as the docs (https://docs.solo.io/gloo-edge/latest/installation/enterprise/) suggest works just fine.  I'm guessing there are some deprecated repo references that when I pull it locally then try to deploy is causing me to try to re-download the deprecated charts and throwing an error.

**To Reproduce**

**helm version**                                                                                                                            
`version.BuildInfo{Version:"v3.4.2", GitCommit:"23dd3af5e19a02d4f4baa5b2f242645a1a3af629", GitTreeState:"clean", GoVersion:"go1.14.13"}`
**skaffold version**
`v1.7.1`

skaffold.yaml
```
apiVersion: skaffold/v1beta11
kind: Config
deploy:
  helm:
    releases:
    - name: gloo
      chartPath: gloo-ee
      namespace: gloo-system
      wait: true
```
1. `helm pull glooe/gloo-ee --untar`
2. `skaffold deploy -f skaffold.yaml`
3. See error:
```
Tags used in deployment:
Starting deploy...
Helm release gloo not installed. Installing...
Getting updates for unmanaged Helm repositories...
...Successfully got an update from the "https://storage.googleapis.com/solo-public-helm" chart repository
...Successfully got an update from the "https://kubernetes-charts.storage.googleapis.com/" chart repository
...Successfully got an update from the "https://kubernetes-charts.storage.googleapis.com/" chart repository
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "glooe" chart repository
...Successfully got an update from the "timescale" chart repository
...Successfully got an update from the "traefik" chart repository
...Successfully got an update from the "bitnami" chart repository
...Successfully got an update from the "stable" chart repository
Update Complete. ⎈Happy Helming!⎈
Saving 3 charts
Downloading gloo from repo https://storage.googleapis.com/solo-public-helm
Downloading prometheus from repo https://kubernetes-charts.storage.googleapis.com/
Already downloaded grafana from repo https://kubernetes-charts.storage.googleapis.com/
Deleting outdated charts
Error: template: gloo-ee/charts/deprecation/templates/NOTES.txt:10:3: executing "gloo-ee/charts/deprecation/templates/NOTES.txt" at <fail "\n*******************************************************\n* This chart location is deprecated. Learn more here: *\n* https://helm.sh/blog/charts-repo-deprecation/       *\n*******************************************************\n">: error calling fail: 
*******************************************************
* This chart location is deprecated. Learn more here: *
* https://helm.sh/blog/charts-repo-deprecation/       *
*******************************************************
```

**Expected behavior**
I expect the helm chart to be installed onto my cluster.

**Additional context**
- Gloo Edge version `1.5.14`
- Kubernetes version `1.18.8`
- Requirements.yaml ( I think the problem may be with `kubernetes-charts.storage.googleapis.com`):
```dependencies:
  - name: gloo
    repository: https://storage.googleapis.com/solo-public-helm
    version: 1.5.14
  - condition: prometheus.enabled
    name: prometheus
    repository: https://kubernetes-charts.storage.googleapis.com/
    tags:
      - monitoring
    version: 9.5.1
  - condition: grafana.defaultInstallationEnabled
    name: grafana
    repository: https://kubernetes-charts.storage.googleapis.com/
    tags:
      - monitoring
    version: 5.0.8
``` 
- Gloo Helm values file (everything was left as default except the following):
```
prometheus:
  enabled: false
grafana:
  defaultInstallationEnabled: false
global:
  extensions:
    extAuth:
      enabled: false
```

**Comment by jgrossmac** [comment]
WARNING: "kubernetes-charts.storage.googleapis.com" is deprecated for "stable" and will be deleted Nov. 13, 2020.
WARNING: You should switch to "https://charts.helm.sh/stable"

**Comment by kdorosh** [comment]
Thanks for reporting @jgrossmac , someone from Solo should be taking a look at this soon.

**Comment by jgrossmac** [comment]
@kdorosh I've confirmed this works as intended:
```dependencies:
- name: gloo
  repository: https://storage.googleapis.com/solo-public-helm
  version: 1.5.14
- condition: prometheus.enabled
  name: prometheus
  repository: https://charts.helm.sh/stable
  tags:
  - monitoring
  version: 9.5.1
- condition: grafana.defaultInstallationEnabled
  name: grafana
  repository: https://charts.helm.sh/stable
  tags:
  - monitoring
  version: 5.0.8
```

**Comment by mlholland** [comment]
It looks like this issue was addressed on master under https://github.com/solo-io/solo-projects/pull/1988. Back-porting those changes (which are the same changes that Jason made, expect in the requirements.yaml template file in solo-projects, plus the same fix in a few other places), should solve the problem.

**Comment by mlholland** [comment]
v1.6.0 is about to release with this fix, and this does not really merit a backport to fix it in 1.5, so I'm going to close this after v1.6 is out.

**Comment by sam-heilbron** [comment]
This change has been released in gloo v1.5.x and v1.6.x. Thanks for bringing this to our attention @jgrossmac !
